html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+<h1>Coding Form!</h1>
+
+<form action="">
+  <label for="email">Email <br />
+  <input type="text" id="email" name="email">
+  </label>
+  <div>
+  <label for="whyCode">Why is coding cool? <br />
+    <select name="codingCool" id="whyCode">
+      <option value="selectOne">Please select one</option>
```

Comment:
For the "default" option we usually leave the `value` attribute set to an empty string